### ISSUE TITLE
Incorrect HTMX translations 

### ISSUE THREAD

**Issue by OmLachake** [AnswerDotAI/fasthtml#254]
I was following the tutorial from Jeremy howard on how to make the Todo list and the problem occurred at this function.
```python
def MakeInput():
    return Input(
        placeholder='Add new Idea',
        id='title',
        hx_swap_oob='true'
    )
```

and then calling this function as,

```python

@route('/')
def post(todo:Todo):
    return todos.insert(todo), MakeInput()

```
The issue with the attribute `hx_swap_oob='true'`  is that when the function is called, it is translated as `hx-swap:oob='true'` and then the intended functionality of htmx actually swapping it does not occur. Insead it adds to the list of todos. Like this : 

![resulting issue](https://github.com/user-attachments/assets/44f83dfc-32f5-400b-9897-f566bfb68647)

The problem seems to be in different scripts as well, I just tried the advanced app and I am having the same issues.

![image](https://github.com/user-attachments/assets/cfaa9937-0e39-4239-a2d6-652a29243229)

The following HTML was returned as the response.

```html
<li id="todo-2">
  <a href="#" hx-get="/toggle/2" hx-target="#todo-2">Toggle</a>
  <a href="#" hx-swap="outerHTML" hx-delete="/2" hx-target="#todo-2">Delete</a>
new todo
</li>

<input placeholder="Add new Idea" hx-swap:oob="true" id="title" name="title">

```

But, when I passed the attributes as the dictionary, 

```python

def MakeInput():
    return Input(
        placeholder='Add new Idea',
        id='title',
        **{'hx-swap-oob': 'true'}
    )

```

It works perfectly. Here is the returned output

```html
<li id="todo-2">
  <a href="#" hx-get="/toggle/2" hx-target="#todo-2">Toggle</a>
  <a href="#" hx-swap="outerHTML" hx-delete="/2" hx-target="#todo-2">Delete</a>
new todo
</li>

<input placeholder="Add new Idea" hx-swap-oob="true" id="title" name="title">
```

You can try it out here :  https://github.com/OmLachake/fast-html-try/blob/main/database.py

**Comment by nibinho** [comment]
Try updating to the newest version, this was fixed in 0.3.1.

**Comment by OmLachake** [comment]
> Try updating to the newest version, this was fixed in 0.3.1.

Thanks. It worked.
